sig-auth/annual-report-2021.md

```diff
@@ -0,0 +1,109 @@
+
+- We do not currently measure this. Need to look at how other SIGs do this today.
+
+**Is there a healthy onboarding and growth path for contributors in your SIG? What are some activities that the group does to encourage this? What programs are you participating in to grow contributors throughout the contributor ladder?**
+
+- Currently there is no onboarding or growth path. This is something we are working on and learning from other SIGs. We will start by creating a CONTRIBUTING.md file.
+
+**What programs do you participate in for new contributors?**
+
+- We have intro sessions at KubeCon to help new contributors get started.
+
+- We now have weekly issues and PRs triage meetings. New contributors can join to help with new issues and PRs. 
+
+**Does the group have contributors from multiple companies/affiliations? Can end users/companies contribute in some way that they currently are not?**
+
+- Yes. Our chairs, leads, contributors, participants, and subproject owners are from various companies.
+
+## Current initiatives and project health
+
+- Please include links to KEPs and other supporting information that will be beneficial to multiple types of community members. 
+
+**What are initiatives that should be highlighted, lauded, shout out, that your group is proud of? Currently underway? What are some of the longer tail projects that your group is working on?**
+
+- [CSR v1](https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/1513-certificate-signing-request)
+- [Token Request / bound SA token admission](https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/1205-bound-service-account-tokens)
+- [client-go auth plugins](https://github.com/kubernetes/enhancements/blob/master/keps/sig-auth/541-external-credential-providers)
+- [external kubelet credential providers](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/2133-kubelet-credential-providers)
+- [New features in Secrets Store CSI driver](https://secrets-store-csi-driver.sigs.k8s.io/introduction.html#features)
+- [PSP v2](https://github.com/kubernetes/enhancements/issues/2579)
+
+**Year to date KEP work review: What’s now stable? Beta? Alpha? Road to alpha?**
+
+- Stable
+    - Certificates API
+    - TokenRequest
+    - TokenRequestProjection
+    - ServiceAccountIssuerDiscovery
+- Beta
+    - CSIServiceAccountToken
+    - RootCAConfigMap
+    - BoundServiceAccountToken
+- Alpha
+    - Hierarchical Namespace Controller
+
+**What areas and/or subprojects does the group need the most help with?**
+
+- PSP v2
+- Audit Logging
+- Audit ID
+- Testing
+- KMS
+    - [KMS-Plugin: Areas for improvement](https://docs.google.com/document/d/1-WHXX_Dh_MNcJb2QJxF0gOAvLjh0fAnc3QrylWdMZJA/edit)
+
+**What's the average open days of a PR and Issue in your group? / what metrics does your group care about and/or measure?**
+
+- We do not currently have this data and will need to get metrics.
```

Comment:
devstats has some data here but I'm not quite sure how to interpret it:

* https://k8s.devstats.cncf.io/d/15/issues-age-by-sig-and-repository-groups?orgId=1&var-period=d7&var-repogroup_name=Kubernetes&var-sig_name=auth&var-kind_name=All&var-prio_name=All
* https://k8s.devstats.cncf.io/d/73/inactive-issues-by-sig?orgId=1&var-sigs=%22auth%22
* https://k8s.devstats.cncf.io/d/70/awaiting-prs-by-sig?orgId=1&var-sigs=%22auth%22